### PR TITLE
like post

### PR DESCRIPTION
**Pull request by NgengChhengkim**

### CODE REVIEW

File: app/assets/javascripts/like_post.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+$(document).on("turbolinks:load ajaxComplete", function(){
```
Comment (yongsokheng) [inline app/assets/javascripts/like_post.js]: dak chol chum khnea trong new_feeds.js sruol, like, save, share, ...

File: app/controllers/like_posts_controller.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
+class LikePostsController < ApplicationController
+  load_and_authorize_resource
+
+  def create
+    @like_post = current_user.like_posts.create like_post_params
```
Comment (yongsokheng) [inline app/controllers/like_posts_controller.rb]: respond_to format ....

File: app/controllers/like_posts_controller.rb [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  end
+
+  def destroy
+    @activity = @like_post.activity
+    @like_post.destroy
```
Comment (yongsokheng) [inline app/controllers/like_posts_controller.rb:14]: respond_to format ...

File: app/views/like_posts/create.js.erb [hunk 4]
```diff
@@ -0,0 +1,4 @@
+<% if @like_post.errors.blank? %>
```
Comment (yongsokheng) [inline app/views/like_posts/create.js.erb]: @like_post.errors.any?

Comment (NgengChhengkim) [reply app/views/like_posts/create.js.erb]: like thanh cong ban oy thuc hien

File: app/views/shared/_button_like.html.erb [hunk 5]
```diff
@@ -0,0 +1,14 @@
+<% like_post = current_user.like_posts.by_activity activity.id %>
+<% if like_post.present? %>
+   <%= link_to "javascript:void(0)", id: "activity-#{activity.id}",
```
Comment (yongsokheng) [inline app/views/shared/_button_like.html.erb]: dat ten la activity-like tov na, ach mean activity chren tiet

File: app/views/like_posts/create.js.erb [hunk 6]
```diff
@@ -0,0 +1,4 @@
...
+  $("#activity-<%= @like_post.activity_id %>")
+    .replaceWith("<%= j render 'shared/button_like', activity: @like_post.activity, type: 'unlike' %>");
```
Comment (yongsokheng) [inline app/views/like_posts/create.js.erb]: type dung o dau?

File: app/views/like_posts/create.js.erb (hunk 4 above)
Comment (yongsokheng) [reply app/views/like_posts/create.js.erb]: ber cheng them if else knong controller, ber ot thanh cong oy flash[:alert]
knong js, ber ot thanh cong oy render flash

Comment (NgengChhengkim) [reply app/views/like_posts/create.js.erb]: ort brer flash. brer error oy chenh red thom da. flash a krok merl

File: app/views/like_posts/create.js.erb [hunk 7]
```diff
@@ -0,0 +1,6 @@
+var activity_like = $("#activity-like-<%= @like_post.try :activity_id %>");
+<% if @like_post.errors.any? %>
```
Comment (yongsokheng) [inline app/views/like_posts/create.js.erb:1]: mech ot brer flash? oy thong nhat doch khnea tov na, merl knong pull nhom mech dae?

File: app/views/like_posts/create.js.erb (hunk 4 above)
Comment (yongsokheng) [reply app/views/like_posts/create.js.erb]: mech deng flash akrok merl? test nov? :smile:

Comment (yongsokheng) [reply app/views/like_posts/create.js.erb]: sak save post muoy tov, chenh flash cheng ta, error chenh cheng dae tae flash :alert

**Review by yongsokhengkhmer — APPROVED** [review]

**Pull request merged**